packages/dataprovider/src/buildVariables.test.ts

```diff
@@ -119,6 +119,58 @@ describe("buildVariables", () => {
         },
       });
     });
+
+    it("returns create new many relations", () => {
+      const params = {
+        data: {
+          email: "albert.einstein@patentamt-bern.ch",
+          firstName: "Albert",
+          lastName: "Einstein",
+          wantsNewsletter: false,
+          roles: [{ name: "admin" }, { name: "user" }],
```

Comment:
its not obvious that this params should create a new record. Is it because they are lacking ids?

if so, maybe we should rearange the test cases and say:

"it creates an entity with newly created related entities when related entities have no ids"
"it creates an entity with connected related entities when related entities have ids"